src/test/kotlin/io/quarkus/code/misc/QuarkusExtensionUtilsTest.kt

```diff
@@ -63,7 +63,7 @@ internal class QuarkusExtensionUtilsTest {
                 category = "Web",
                 tags = listOf(),
                 default = false,
-                keywords = listOf(),
+                keywords = listOf("blocking", "client", "foundation", "framework", "i", "layer", "netty", "non", "o", "quarkus", "server", "used"),
```

Comment:
I would also filter out words that are shorter than 3 chars. It would reduce size of the `stopWords` list and I think that the most important words will have 3 or more chars, wdyt?

Reply:
OK, let me do that.